types/zepto/index.d.ts

```diff
@@ -1316,12 +1316,12 @@ interface ZeptoCollection {
 	* @param fn
 	* @return
 	**/
-	off(type: string, selector: string, fn: (e: Event) => boolean): ZeptoCollection;
+	off(type: string, selector: string, fn: ZeptoEventHandler): ZeptoCollection;
```

Comment:
But this means you can return a non-boolean back - I can't judge how desirable this is.

Reply:
https://github.com/madrobby/zepto/blob/763b3d6dc3b4350759ed30aa196cd2b6e39efcfb/src/event.js#L69

only ```false``` have special meaning, all other is ignored, so i think any  is ok...